scripts/examples/android_app_as.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
what is the difference and relation between this and android_app.sh?

Could we update that instead?

in #8862 I am currently updating the build_examples.py to be able to build the android app (and also makes the build in 'out'  rather than in-place.

Could we use that script instead of adding more shell scripts? I am concerned that our compilation logic is reasonably inconsistent across shell files.

Reply:
The main different is android_app_as.sh added "--ide=json --json-ide-script=//scripts/examples/gn_to_cmake.py" and fixed output founder to include in build.gradle. I understand you point add will try it

Reply:
update at bcecb22bdef7060af2a6e5012cf6aa403dfe8873
https://github.com/project-chip/connectedhomeip/pull/8875/commits/b575d5b18bde7e2c0ef34d531f034e6432c6de7f

Reply:
Hello @andy31415, any update on this? Thanks